content/blog/2020-04-usethis-1-6-0/index.Rmarkdown

```diff
@@ -39,15 +39,45 @@ This release also includes a handful of new functions (my favourite is [`rename_
 
 ## GitHub Actions
 
-[GitHub Actions](https://github.com/features/actions) allows you to automatically run code whenever you push to GitHub. If you're developing a package this allows you to automate tasks like:
+[GitHub Actions](https://github.com/features/actions) is a continuous integration service that allows you to automatically run code whenever you push to GitHub. If you're developing a package this allows you to automate tasks like:
 
-* Run `R CMD check` on multiple platforms (Linux, Windows, and Mac).
+* Run `R CMD check` on multiple platforms (Linux, Windows, and Mac). 
 * Record the code coverage of your unit tests.
 * Re-build your [pkgdown](https://pkgdown.r-lib.org/) website.
 
-Outside of a package, you can also use them to rebuild blogdown and bookdown sites, or regularly re-knit `.Rmd` files. 
+Outside of a package, you can also use them to rebuild [blogdown](https://github.com/r-lib/actions/blob/master/examples/blogdown.yaml) and [bookdown](https://github.com/r-lib/actions/blob/master/examples/bookdown.yaml) sites, or regularly re-knit `.Rmd` files. 
 
-Learn more by reading [_Github Actions with R_](https://ropenscilabs.github.io/actions_sandbox/), by Chris Brown, Murray Cadzow, Paula A Martinez, Rhydwyn McGuire, David Neuzerling, David Wilkinson, and Saras Windecker, or watching Jim Hester's [rstudio::conf(2020) presentation](https://resources.rstudio.com/rstudio-conf-2020/azure-pipelines-and-github-actions-jim-hester).
+A GitHub action is just a yaml file stored in `.github/workflows`, and we bundle a number of helpers to create these files:
```

Comment:
Technically this should be `A GitHub Action workflow is just a yaml file`...

A GitHub Action is the docker or JS based things you use _in_ the workflows